mkt/extensions/validation.py

```diff
@@ -19,6 +19,8 @@ class ExtensionValidator(object):
     error.
     """
     errors = {
+        'AUTHOR_TOO_LONG': _(
+            u'The `author` property cannot be longer than 255 characters.'),
```

Comment:
Updated to 128. A little weird since description is at 132 though, but I guess that's fine ? I also added a bunch of validation tests.